src/lib/components/media/landscape-card.svelte

```
@@ -116,7 +116,7 @@
                             {:then ratingsData}
                                 {#if ratingsData?.scores?.length}
                                     {#each ratingsData.scores as score (score.name)}
-                                        <!-- svelte-ignore svelte/no-navigation-without-resolve -->
+                                        <!-- svelte-ignore a11y_no_static_element_interactions -->
```

ESLint

[error] svelte-ignore comment is used, but not warned

(svelte/no-unused-svelte-ignore)